html/admin/project/crew/vacantCrew.twig

```diff
@@ -151,28 +144,50 @@
                             </div>
                             <div class="input-group" style="margin-bottom: 5px;">
                                 <div class="input-group-prepend">
-                                    <span class="input-group-text">Collect Phone Numbers</span>
+                                    <span class="input-group-text" data-toggle="tooltip" data-placement="bottom" title="Allow people to assign themselves to this role, and automatically close the role once it has been filled">
+                                        First Come First Served
+                                    </span>
                                 </div>
-                                <input type="checkbox" class="form-control" name="projectsVacantRoles_collectPhone" {{ role. projectsVacantRoles_collectPhone == 1 or role.NEW ? 'checked' : ''  }}  />
+                                <input type="checkbox" class="form-control" name="projectsVacantRoles_firstComeFirstServed" {{ role.projectsVacantRoles_firstComeFirstServed == 1 ? 'checked' : ''  }}  />
                             </div>
+
+                            <hr/>
+                            <h3>Role Visibility</h3>
                             <div class="input-group" style="margin-bottom: 5px;">
                                 <div class="input-group-prepend">
-                                    <span class="input-group-text">Roles available</span>
+                                    <span class="input-group-text">Visible on Crew Vacancies List</span>
                                 </div>
-                                <input type="number" class="form-control" required name="projectsVacantRoles_slots" min="1" value="{{ role.projectsVacantRoles_slots }}" />
+                                <input type="checkbox" class="form-control" name="projectsVacantRoles_open" {{ role.projectsVacantRoles_open == 1 or role.NEW ? 'checked' : ''  }}  />
+                            </div>
+                            <div class="input-group" style="margin-bottom: 5px;">
+                                <div class="input-group-prepend">
+                                    <span class="input-group-text">Visible on Public Site</span>
+                                </div>
+                                <input type="checkbox" class="form-control" name="projectsVacantRoles_showPublic" {% if publicData.enabled != true %}disabled title="Public Site is Disabled"{% elseif role.projectsVacantRoles_visibleToGroups != null %}disabled title="Role is only available to specified groups"{% elseif page.projectsVacantRoles_showPublic == 1 or page.NEW %}checked{% endif %} />
```

Comment:
Because of the way round this if statement is, my hunch is that if the public site is disabled and the json is null, this will not disable the checkbox

Reply:
I've tried a few setting combinations, and It seems to be working fine for me? Happy to swap it round but not found an issue so far